src/game_player.cpp

```diff
@@ -269,7 +269,8 @@ void Game_Player::Update(bool process_movement) {
 			}
 			SetMoveSpeed(vehicle->GetMoveSpeed());
 			vehicle->SetDirection(GetDirection());
-			vehicle->SetSpriteDirection(GetSpriteDirection());
+			vehicle->SetSpriteDirection(Left);
+			SetSpriteDirection(Left);
```

Comment:
> Fix directions when finish boarding boat ship 

Same problem here with fixed graphics I suppose?

Reply:
Flag is ignored, just like airship disembark.